numpy/_core/memmap.py

```diff
@@ -339,6 +339,59 @@ def flush(self):
         if self.base is not None and hasattr(self.base, 'flush'):
             self.base.flush()
 
+    def madvise(self, option):
+        """
+        Give advice about the memory region corresponding to this array.
+
+        Parameters
+        ----------
+        option : int
+            One of the `madvise` options, e.g., `mmap.MADV_NORMAL`,
+            `mmap.MADV_RANDOM`, `mmap.MADV_SEQUENTIAL`, `mmap.MADV_WILLNEED`,
+            `mmap.MADV_DONTNEED`, `mmap.MADV_FREE`, `mmap.MADV_REMOVE`,
+            `mmap.MADV_DONTFORK`, `mmap.MADV_DOFORK`, `mmap.MADV_MERGEABLE`,
+            `mmap.MADV_UNMERGEABLE`, `mmap.MADV_HUGEPAGE`,
+            `mmap.MADV_NOHUGEPAGE`, `mmap.MADV_SOFT_OFFLINE`,
+            `mmap.MADV_COLD`, `mmap.MADV_PAGEOUT`, `mmap.MADV_POPULATE`.
```

Comment:
Rather than listing them all, maybe just link to `mmap.mmap.madvise`? I am confused about the `madvise` not failing the doc build CI.